models/GAN/Discriminator.py

```diff
@@ -21,35 +21,27 @@ class Discriminator(nn.Module):
     def __init__(self, batch_momentum=0.1, padding_mode=PaddingMode.REFLECT):
         super().__init__()
 
-        sequence = []
-        filter_sizes = [64, 128, 256, 512]
-        prev_out_channels = 3
-
-        for n_filters in filter_sizes:
-
-            # skip norm layer for first conv block
-            norm_type = NormType.INSTANCE if n_filters != 64 else NormType.NONE
-            stride = 2 if n_filters != 512 else 1
-
-            sequence += [
-                Conv2DBlock(in_channels=prev_out_channels, stride=stride,
-                            out_channels=n_filters, kernel_size=4,
-                            activation_type=ActivationType.LEAKY,
-                            padding_mode=padding_mode, norm_type=norm_type,
-                            batch_momentum=batch_momentum)
-            ]
-
-            prev_out_channels = n_filters
-
-        # convolve result into one-dimensional output
-        sequence += [
-            Conv2DBlock(in_channels=prev_out_channels, out_channels=1,
+        def patchGANConvLayer(in_channels, out_channels, stride, norm_type):
```

Comment:
The for loop was a bit arbitrary and hard to understand. Since it's always 4 layers I figured it'd be easier to just write them out.